model/src/main/kotlin/config/Excludes.kt

```diff
@@ -36,4 +41,32 @@ data class Excludes(
          */
         @JsonInclude(JsonInclude.Include.NON_EMPTY)
         val scopes: List<ScopeExclude> = emptyList()
-)
+) {
+    /**
+     * Return the [ProjectExclude] for this [project], or null if there is none.
```

Comment:
Nit: Maybe drop "this" as it could be misread to say `this` is a `Project`, whereas it's an `Excludes` object.

Reply:
Done.